src/particle_rigidbody/4C_particle_rigidbody_result_test.cpp

```diff
@@ -12,6 +12,8 @@
 #include "4C_particle_rigidbody_interface.hpp"
 #include "4C_utils_exceptions.hpp"
 
+#include <algorithm>
```

Comment:
This change is unrelated to the rest of the pull request but without this include main did not compile for me.

Reply:
Out of curiosity: which compiler are you using?

Reply:
gcc-14.2.0